lib/requests/diaryFoods.js

```diff
@@ -12,6 +12,7 @@ const getMeals = () => {
       meals.forEach(meal => populateMealTable(meal))
     })
     .then(meals => calculateTotalCalories())
+    .then(() => populateRemainingCalories())
```

Comment:
I like that you added this after the total calories have been calculated to make sure that it gets rendered to the page.